### PR TITLE
Enable file header analyzer (IDE0073) in build

### PR DESCRIPTION
**Pull request by mavasani**
First commit moves us to a newer CodeStyle NuGet package and enables IDE0073 as a warning.
Second commit was result of a Fix all in solution operation for its code fix. No manual code changes were done. I skimmed through each changed file to validate the fix did the appropriate change.

Thanks to @sharwell for quick turnaround in implementing this analyzer and moving it to CodeStyle NuGet package for CI support.

### CODE REVIEW

**Comment by sharwell** [comment]
@mavasani I'd like to fix a few bugs I see from this pull request, and then resubmit it so we can see they are fixed before merging

**Comment by mavasani** [comment]
@sharwell Sounds good. I will wait for your signoff on this PR before merge.

File: src/Analyzers/VisualBasic/Tests/RemoveUnnecessaryImports/RemoveUnnecessaryImportsTests_FixAllTests.vb [hunk 1]
```diff
@@ -1,4 +1,8 @@
-﻿Option Strict Off
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Option Strict Off
 ' Licensed to the .NET Foundation under one or more agreements.
 ' The .NET Foundation licenses this file to you under the MIT license.
 ' See the LICENSE file in the project root for more information.
```
Comment (sharwell) [inline src/Analyzers/VisualBasic/Tests/RemoveUnnecessaryImports/RemoveUnnecessaryImportsTests_FixAllTests.vb]: 📝 This one should be manually corrected.

File: src/EditorFeatures/VisualBasicTest/Diagnostics/GenerateType/GenerateTypeTests_Dialog.vb [hunk 2]
```diff
@@ -1,4 +1,8 @@
-﻿Option Strict Off
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Option Strict Off
 ' Licensed to the .NET Foundation under one or more agreements.
 ' The .NET Foundation licenses this file to you under the MIT license.
 ' See the LICENSE file in the project root for more information.
```
Comment (sharwell) [inline src/EditorFeatures/VisualBasicTest/Diagnostics/GenerateType/GenerateTypeTests_Dialog.vb]: 📝 This one should be manually corrected

**Pull request merged**